vcbuild.bat

```diff
@@ -122,6 +122,9 @@ if defined build_release (
 
 :: assign path to node_exe
 set "node_exe=%config%\node.exe"
+set "node_gyp_exe="%node_exe%" deps\npm\node_modules\node-gyp\bin\node-gyp"
+if "%target_env%"=="vs2015" set "node_gyp_exe=%node_gyp_exe% --msvs_version=2015"
+if "%target_env%"=="vs2017" set "node_gyp_exe=%node_gyp_exe% --msvs_version=2017"
```

Comment:
I had the same thought, but if `vs20XX` was specified explicitly, only that version will be tried, so it's ok to set early.